doc/modules/ROOT/pages/index.adoc

```diff
@@ -158,6 +167,17 @@ const client = await Client.newHazelcastClient(clientConfig);
 ----
 --
 
+Python::
++
+--
+[source, python]
+----
+client = hazelcast.HazelcastClient(
```

Comment:
If we change above with the actual way of installing the client, we may add the import statement here and make it like

```python
import hazelcast

client = hazelcast.HazelcastClient(
    cluster_members=["hz-hazelcast"],
)
```

Reply:
done